### PR TITLE
Updates May 25th

### PR DESCRIPTION
**Pull request by yschimke**
#### WHAT

Updates May 25th

#### WHY

Routine bumps

#### Checklist :clipboard:
- [x] Add explicit visibility modifier and explicit return types for public declarations
- [x] Run spotless check
- [x] Run tests
- [x] Update metalava's signature text files

### CODE REVIEW

File: media/ui/src/test/java/com/google/android/horologist/media/ui/screens/browse/PlaylistDownloadBrowseScreenA11yTallScreenshotTest.kt [hunk 1]
```diff
@@ -19,19 +19,22 @@
...
 import org.robolectric.annotation.Config
 
 @Config(
     sdk = [30],
     qualifiers = "w227dp-h400dp-small-notlong-notround-watch-xhdpi-keyshidden-nonav"
 )
-class PlaylistDownloadBrowseScreenA11yTallScreenshotTest : ScreenshotBaseTest() {
+class PlaylistDownloadBrowseScreenA11yTallScreenshotTest : ScreenshotBaseTest(
+    screenshotTestRuleParams {
+    }
```
Comment (luizgrp) [inline media/ui/src/test/java/com/google/android/horologist/media/ui/screens/browse/PlaylistDownloadBrowseScreenA11yTallScreenshotTest.kt:36]: unnecessary empty constructor, did you mean to add `enableA11y = true`?

**Review by luizgrp — APPROVED** [review]

File: media/ui/src/test/java/com/google/android/horologist/media/ui/screens/browse/PlaylistDownloadBrowseScreenA11yTallScreenshotTest.kt (hunk 1 above)
Comment (yschimke) [reply media/ui/src/test/java/com/google/android/horologist/media/ui/screens/browse/PlaylistDownloadBrowseScreenA11yTallScreenshotTest.kt:36]: It's a bit fiddly to flip this back and forth, so I find this helpful.

But really I should fix the record mode!

**Comment by yschimke** [comment]
@luizgrp blocked by https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin/issues/907

Should I

1) disable health plugin?
2) change all dependencies to the -android one
3) Define a bunch of overrides? https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin/issues/907

File: auth/composables/src/test/java/com/google/android/horologist/test/toolbox/PositionedState.kt [hunk 2]
```diff
@@ -17,16 +17,31 @@
...
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.unit.dp
 import com.google.android.horologist.compose.layout.ScalingLazyColumnDefaults
 import com.google.android.horologist.compose.layout.ScalingLazyColumnState
-import com.google.android.horologist.compose.tools.a11y.forceState
+import kotlinx.coroutines.launch
 
 @Composable
 fun positionedState(
```
[the remark this answers is not in the document]
Comment (luizgrp) [reply auth/composables/src/test/java/com/google/android/horologist/test/toolbox/PositionedState.kt:29]: https://github.com/google/horologist/issues/1313

**Comment by luizgrp** [comment]
@yschimke happy to disable the plugin until is fixed, we can raise an issue to track

**Pull request merged**